README.md

```diff
@@ -73,9 +73,6 @@ the configuration options described in
   will receive rewards. The value must be a mainnet address starting with
   `f410`, `0x`.
 
-  Addresses starting with `f1` are deprecated. We allow them for now but will
-  start rejecting them in an upcoming version.
-
```

Comment:
Maybe we can keep the mention of f1 addresses and explain why we don't support them.

> Addresses starting with `f1` are not allowed. Rewards for Station operators are administered by a FEVM smart contract. It is currently technically complex to make payments to `f1` addresses.

Reply:
good idea!